packages/admin-ui/src/Select/Select.tsx

```diff
@@ -0,0 +1,503 @@
+import * as React from "react";
+import { ReactComponent as ChevronUp } from "@material-design-icons/svg/outlined/keyboard_arrow_up.svg";
+import { ReactComponent as ChevronDown } from "@material-design-icons/svg/outlined/keyboard_arrow_down.svg";
+import { ReactComponent as Check } from "@material-design-icons/svg/outlined/check.svg";
+import { ReactComponent as Close } from "@material-design-icons/svg/outlined/close.svg";
+import { makeDecoratable } from "@webiny/react-composition";
+import * as SelectPrimitive from "@radix-ui/react-select";
+import { cva, type VariantProps } from "class-variance-authority";
+import { cn } from "~/utils";
+import { useSelect } from "./useSelect";
+import { SelectOptionDto } from "./SelectOptionDto";
+import { SelectOptionFormatted } from "./SelectOptionFormatted";
+
+const SelectRoot = SelectPrimitive.Root;
+
+const SelectGroup = SelectPrimitive.Group;
+
+const SelectValue = SelectPrimitive.Value;
+
+/**
+ * Icon
+ */
+interface IconWrapperProps {
+    icon: React.ReactElement;
+}
+
+const Icon = ({ icon }: IconWrapperProps) => {
+    return (
+        <SelectPrimitive.Icon asChild className={"h-4 w-4"}>
```

Comment:
We'll probably need to back here, and use tokens instead of h-4 w-4.

I see the Icon component is not ready for dev in Figma, so I'll ping Kreso about it and see if we can wrap that one up.